src/Exercise10/Exercise10SelfCheckout.java

```diff
@@ -6,7 +6,6 @@
 
 
     public int subtotalCalculator(Item... items) {
```

Comment:
This logic is duplicated in several places, so it's a prime candidate to be pulled out into its own method and simply called from the other ones.